lib/webpacker/helper.rb

```diff
@@ -141,8 +141,6 @@ def preload_pack_asset(name, **options)
   #   <%= stylesheet_pack_tag 'calendar' %>
   #   <%= stylesheet_pack_tag 'map' %>
   def stylesheet_pack_tag(*names, **options)
-    return "" if Webpacker.inlining_css?
```

Comment:
Removing this line definitely breaks HMR with CSS modules.

If somebody doesn't want HMR, the CSS is exported. So this is a doc issue if anything.

Reply:
Yeah, this PR will not help with that issue. It would be great to restore the behavior from the previous version of webpacker when HMR works with `stylesheet_pack_tag`